bin/node/runtime/src/lib.rs

```diff
@@ -999,15 +1000,15 @@ parameter_types! {
 	pub const DesiredRunnersUp: u32 = 7;
 	pub const MaxVoters: u32 = 10 * 1000;
 	pub const MaxCandidates: u32 = 1000;
-	pub const ElectionsPhragmenPalletId: LockIdentifier = *b"phrelect";
+	pub const ElectionsPalletId: LockIdentifier = *b"phrelect";
```

Comment:
a note about why this name is kept the same even though the pallet name changed.

Reply:
Do you suggest this note to be added as a comment in the code or elsewhere? (I will add it in the PR description too)

Reply:
I added the comment on this and in the companion PR as well.